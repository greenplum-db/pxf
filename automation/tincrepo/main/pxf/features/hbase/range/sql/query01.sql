-- @description query01 for PXF HBase - range filter

SELECT * from pxf_hbase_table WHERE "cf1:q3" > '00000090' AND "cf1:q3" <= '00000103' ORDER BY recordkey;