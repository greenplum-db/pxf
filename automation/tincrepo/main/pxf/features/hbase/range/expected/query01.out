-- @description query01 for PXF HBase - range filter
SET datestyle TO 'ISO, MDY';
SELECT * from pxf_hbase_table WHERE "cf1:q3" > '00000090' AND "cf1:q3" <= '00000103' ORDER BY recordkey;
 recordkey |    cf1:q1     |          cf1:q2          | cf1:q3 |     cf1:q4     | cf1:q5 |   cf1:q6    | cf1:q7 | cf1:q8 |      cf1:q9       | cf1:q10 |                                               cf1:q11                                                |         q12         
-----------+---------------+--------------------------+--------+----------------+--------+-------------+--------+--------+-------------------+---------+------------------------------------------------------------------------------------------------------+---------------------
 00000090  | ASCII00000090 | UTF8_計算機用語_00000090 |     91 | lookup00000180 |   90.9 |  90909090.9 | {      |     90 |  7290000000000090 | t       |          1000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000 | 1970-01-01 00:09:00
 00000091  | ASCII00000091 | UTF8_計算機用語_00000091 |     92 | lookup00000182 |  91.91 | 91919191.91 | |      |     91 | -7535710000000091 | f       |         10000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000 | 1970-01-01 00:09:06
 00000092  | ASCII00000092 | UTF8_計算機用語_00000092 |     93 | lookup00000184 |  92.92 | 92929292.92 | }      |     92 |  7786880000000092 | t       |        100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000 | 1970-01-01 00:09:12
 00000093  | ASCII00000093 | UTF8_計算機用語_00000093 |     94 | lookup00000186 |  93.93 | 93939393.93 | ~      |     93 | -8043570000000093 | f       |       1000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000 | 1970-01-01 00:09:18
 00000094  | ASCII00000094 | UTF8_計算機用語_00000094 |     95 | lookup00000188 |  94.94 | 94949494.94 |        |     94 |  8305840000000094 | t       |      10000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000 | 1970-01-01 00:09:24
 00000095  | ASCII00000095 | UTF8_計算機用語_00000095 |     96 | lookup00000190 |  95.95 | 95959595.95 | !      |     95 | -8573750000000095 | f       |     100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000 | 1970-01-01 00:09:30
 00000096  | ASCII00000096 | UTF8_計算機用語_00000096 |     97 | lookup00000192 |  96.96 | 96969696.96 | "      |     96 |  8847360000000096 | t       |    1000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000 | 1970-01-01 00:09:36
 00000097  | ASCII00000097 | UTF8_計算機用語_00000097 |     98 | lookup00000194 |  97.97 | 97979797.97 | #      |     97 | -9126730000000097 | f       |   10000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000 | 1970-01-01 00:09:42
 00000098  | ASCII00000098 | UTF8_計算機用語_00000098 |     99 | lookup00000196 |  98.98 | 98989898.98 | $      |     98 |  9411920000000098 | t       |  100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000 | 1970-01-01 00:09:48
 00000099  | ASCII00000099 | UTF8_計算機用語_00000099 |    100 | lookup00000198 |  99.99 | 99999999.99 | %      |     99 | -9702990000000099 | f       | 1000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000 | 1970-01-01 00:09:54
(10 rows)

